src/radical/pilot/agent/scheduler/base.py

```diff
@@ -241,6 +241,10 @@ def initialize_child(self):
         # during agent startup.  We dig them out of the config at this point.
         #
         # NOTE: this information is insufficient for the torus scheduler!
+
+        # MING: What is lm info? All the other things make sense, but lm_info
+        #       is a bit unclear.
+
```

Comment:
This is documented here: https://github.com/radical-cybertools/radical.pilot/blob/feature/gpu/src/radical/pilot/agent/scheduler/base.py#L113  Would you mind adding a link/reference?

Reply:
I would to distinguish between `lrms_info` and `lm_info`. `lrms_info` provides info on the nodes available and the number of CPU/GPU cores per node. Within `lrms_info` is something called `lm_info`. I would like to know more about `lm_info`. The only documentation on this [here](https://github.com/radical-cybertools/radical.pilot/blob/feature/gpu/src/radical/pilot/agent/scheduler/base.py#L129). `lm_info` is assigned to `self._lrms_lm_info` during initialization. This information is passed around in `continuous.py` when slots are defined, but nothing is done to `lm_info`. What is it for?